runtime/oti/j9.h

```diff
@@ -331,7 +331,8 @@ static const struct { \
 
 #if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
 #define J9CLASS_UNPADDED_INSTANCE_SIZE(clazz) J9_VALUETYPE_FLATTENED_SIZE(clazz)
-/* TODO replace with J9_IS_J9CLASS_ALLOW_DEFAULT_VALUE(clazz) J9_ARE_ALL_BITS_SET((clazz)->classFlags, J9ClassAllowsInitialDefaultValue)*/
+#define J9_IS_J9CLASS_ALLOW_DEFAULT_VALUE(clazz) J9_ARE_ALL_BITS_SET((clazz)->classFlags, J9ClassAllowsInitialDefaultValue)
+/* TODO replace with J9_IS_J9CLASS_ALLOW_DEFAULT_VALUE */
```

Comment:
Please explain (or remove) this comment.

Reply:
This was a reminder from before the q types removal work had been started https://github.com/eclipse-openj9/openj9/issues/18157. It would be removed when the J9_IS_J9CLASS_PRIMITIVE_VALUETYPE macro goes away but I don't think its really needed.